cmd/login.go

```diff
@@ -102,208 +112,157 @@ type deviceTokenResponse struct {
 
 // If we already have a refresh token, we don't need to go through the whole device code
 // interaction. We can simply use the refresh token to request another access token.
-func refreshAccessToken() (accessToken string, err error) {
+func getFreshAccessToken() (accessToken string) {
```

Comment:
This rename is slightly incorrect, as we only get a fresh access token if the current one is expired rather than every time this function is called.

Reply:
Changed to getAccessToken, could also be getActiveAccessToken.